docs/basics/firststeps.md

```diff
@@ -20,11 +20,15 @@ To delete the current annotation in progress, click the **Clear Annotation** but
 To select a different annotation click the **Select** button, click an area of the desired annotation and select the desired functionality to alter.
 To undo the last action, click the **Undo last action** and to redo the action, click the **Redo last action**.
 
-Once annotations have been completed, click the **Save Annotations** button to confirm your changes.
-Please note: if you leave annotate without saving then your annotations will not be added to the image metadata.
```

Comment:
Maybe keep line 24 in for the time being? The saving bug has not been fixed yet (I believe).